lasagne/init.py

```diff
@@ -132,6 +279,16 @@ def sample(self, shape):
 
 
 class Sparse(Initializer):
+    """Initialize weights as sparse matrix.
+
+    Parameters
+    ----------
+    sparsity : float
+        Fraction of weights with non-zero value. Larger value gives less
+        sparsity.
```

Comment:
Fraction of weights with non-zero value. Larger value gives less sparsity. The given fraction will be matched exactly for each column of the matrix.?